pages/common/pylint.md

```diff
@@ -0,0 +1,15 @@
+# pylint
+
+> A Python code linter.
+> More information: <https://pylint.pycqa.org/en/latest/>.
+- Show lint errors in a file:
+
+`pylint file`
+
+- Use the configuration from a configuration file, usually named `pylintrc`:
+
+`pylint --rcfile ~/path/to/pylintrc file`
+
+- Disable a specific error code:
+
+`pylint -d error_code file`
```

Comment:
I think we should give an example here
```suggestion
`pylint --disable {{C,W,no-error,design}} {{path/to/file}}`
```

Reply:
Totally agree, it's much clearer that way.